metricbeat/module/nginx/stubstatus/stubstatus.go

```diff
@@ -0,0 +1,118 @@
+// Package stubstatus reads server status from nginx host under /server-status, ngx_http_stub_status_module is required.
+package stubstatus
+
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/logp"
+	"github.com/elastic/beats/metricbeat/mb"
+)
+
+const (
+	// defaultScheme is the default scheme to use when it is not specified in
+	// the host config.
+	defaultScheme = "http"
+
+	// defaultPath is the default path to the mod_status endpoint on the
+	// Apache HTTPD server.
+	defaultPath = "/server-status"
+)
+
+var (
+	debugf = logp.MakeDebug("apache-status")
```

Comment:
`s/apache-status/nginx-stubstatus/` or maybe just `nginx`. Not sure how much granularity we need.